modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectField/EditObjectField.tsx

```diff
@@ -70,6 +68,10 @@ export default function EditObjectField({
 	readOnly,
 	workflowStatuses,
 }: EditObjectFieldProps) {
+	const [dbObjectFieldRequired, setDbObjectFieldRequired] = useState<
+		boolean
+	>();
+
```

Comment:
Hey @igor-franca, since we will only have the declaration of this useState, you can delete one of them